# RNA sequencing 
 

## calcnormfactors operator

##### Description
`calcnormfactors` operator calculates a normalization factor per column (i.e sample).

##### Usage
Input projection|.
:---|:---
`row`   | represents the genes
`col`   | represents the samples
`y-axis`             | is the input data for which to compute the normalization factor


Output relations|.
:---|:---
`normfactor`| numeric, per column (i.e. per sample)

##### Details
The operator is the calcnormfactor in the edgeR bioconductor.


##### References
see the `edgeR` R package for the documentation.


##### See Also


##### Examples


##### GitHub link

[calcnormfactors_operator on GitHub](https://github.com/tercen/calcnormfactors_operator) 

## DESeq2_two_conditions operator

##### Description

`DESeq2_two_conditions` tests for differential gene expression in samples from two conditions using the `DESeq2` package from BioConductor (Love, et al, Genome Biology, 2014).

##### Usage

| Input projection | Description                      |
| :---------------- | :-------------------------------- |
| `row`            | Gene name/identifier             |
| `column`         | Sample name/identifier           |
| `color`          | Represents the groups to compare |
| `y-axis`         | Sequence counts                  |

| Input parameters | Description                                                                              |
| :-----------------| :---------------------------------------------------------------------------------------- |
| `alpha`          | Numeric, adjusted _p value_ cutoff for independent filtering (default = 0.1)             |
| `LFC_shrinkage`  | Logical, whether the returned log fold-change values should be shrinked (default = TRUE) |
| `shrinkage_type` | "normal", "apeglm" or "ashr", Type of shrinkage estimator to use (default = "normal")    |

|  Output relations  | Description                                                                |
| :------------------ | :-------------------------------------------------------------------------- |
| `pvalue`           | numeric, p-value calculated per gene                                       |
| `padj`             | numeric, p-value calculated per gene after adjusting for multiple testing  |
| `baseMean`         | numeric, mean of normalized counts for all the samples                     |
| `log2FoldChange`   | numeric, shrunken log2 fold-change between the two groups to compare       |
| `minus_log10_padj` | numeric, negative log10 transformation of padj for more intuitive plotting |

##### Details

The operator uses the `DESeq2` package from BioConductor.

##### References

Love MI, Huber W, Anders S (2014). “Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2.” Genome Biology, 15, 550. doi: 10.1186/s13059-014-0550-8.

See ["Analyzing RNA-seq data with DESeq2"](https://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html) for further information on DESeq2 by Love, et al.



##### GitHub link

[DESeq2_two_conditions_operator on GitHub](https://github.com/tercen/DESeq2_two_conditions_operator) 

## diffcyt operator

##### Description
`diffcyt` operator a differential analysis of flow cyto data and indicates which marker and cluster combinations are relevant.

##### Usage
Input projection|.
:---|:---
`col`    | `group_id`, `patient_id`
`row`    | `marker_class`, `marker_name`
`y-axis` | values representing measurement


Input parameters|.
:---|:---
`analysis_type` | can be either `DA` (Differential Abundance) or `DS` (Differential State)

Output relations|.
:---|:---
`cluster_id`| character, cluster  name, per cluster, DA and DS output
`LogFC`     | numeric, log fold change, per cluster, DA
`LR`        | numeric, lr, per cluster, DA output
`p_val`     | numeric, p value, per cluster, DA output
`p_adj`     | numeric, adjusted p value, per cluster, DA output
`LogFC`     | numeric, log fold change, per cluster-marker, DS output
`p_val`     | numeric, p value, per cluster-marker, DS output
`p_adj`     | numeric, adjusted p value, per cluster-marker, DS output
`t`         | numeric, t, per cluster-marker, DS output
`B`         | numeric, B, per cluster-marker, DS output
`AvgExp`    | numeric, adjusted p value, per cluster-marker, DS output

##### Details

Performs differential analysis (abundance or state). See the `diffcyt::diffcyt` function in the Bioconductor R pacakge.

##### References
see the github for documentation,
https://github.com/lmweber/diffcyt


##### See Also
[t-test](https://github.com/lmweber/ttest_operator), [anova](https://github.com/tercen/anova_operator), [rfImp](https://github.com/tercen/rfImp_operator)

##### Examples


##### GitHub link

[diffcyt_operator on GitHub](https://github.com/tercen/diffcyt_operator) 

## fgsea operator

##### Description
`fgsea` operator performs a fast gene set enrichment analysis.

##### Usage

Input projection|.
:---|:---
`y-axis`        | numeric, input data usually a ranking statistics, per cell

Output relations|.
:---|:---
`NES`         | numeric, fgsea of the input data
`padj`        | numeric, adjusted p-value

##### Details

The operator takes all the values of a cell and returns the value which is the fgsea. The computation is done per cell. There is one value returned for each of the input cell.

##### References


##### See Also


##### Examples


##### GitHub link

[fgsea_operator on GitHub](https://github.com/tercen/fgsea_operator) 

## FlowSOM tuning operator

##### Description

`flowsomtuning` operator performs flowSOM clustering for different numbers of clusters.

##### Usage

Input projection|.
:---|:---
`row`   | represents the variables (e.g. channels, markers)
`col`   | represents the clusters (e.g. cells) 
`y-axis`| is the value of measurement signal of the channel/marker

Input parameters|.
:---|:---
`min_cluster_number`   | Minimum number of clusters to make
`max_cluster_number`   | Maximal number of clusters to make
`transform`| Transform data?
`seed`   | Random seed

Output relations|.
:---|:---
`cluster`| character, cluster label

##### Details

The operator is a wrapper for the `FlowSOM` function of the `FlowSOM` R/Bioconductor package.

##### References

https://bioconductor.org/packages/FlowSOM/

##### See Also

[flowsom_operator](https://github.com/tercen/flowsom_operator)

[flowsom_mst_shiny_operator](https://github.com/tercen/flowsom_mst_shiny_operator)


##### GitHub link

[flowsomtuning_operator on GitHub](https://github.com/tercen/flowsomtuning_operator) 

## umap operator

##### Description
`umap` operator performs umap analysis.

##### Usage
Input projection|.
:---|:---
`row`   | represents the variables (e.g. genes, channels, markers)
`col`   | represents the observations (e.g. cells, samples, individuals) 
`y-axis`| measurement value


Input parameters|.
:---|:---
`init`           | character, type of initialization for the coordinates, see details
`scale`          | numeric, type of scaling to apply to data
`spread`         | numeric, the effective scale of embedded points. In combination with `min_dist`, this determines how clustered/clumped the embedded points are
`min_dist`       | numeric, the effective minimum distance between embedded point 
`pca`            | numeric, If set to a positive integer value, reduce data to this number of columns using PCA



Output relations|.
:---|:---
`umap01, umap02`| first two components containing the new projected values


##### Details

The operator performs umap analysis. It reduces the amount of variables (i.e. indicated by rows) to a lower number (default 2). This operators wraps the `uwot::umap()`. See (https://github.com/jlmelville/uwot) for more details, especially settings and examples.


##### Reference

##### See Also

[pca](https://github.com/tercen/pca_operator), [tsne](https://github.com/tercen/tsne_operator)

##### Examples



##### GitHub link

[umap_operator on GitHub](https://github.com/tercen/umap_operator) 

## vsn operator

##### Description
`vsn` operator performs a normalization factor per column (i.e. sample).

##### Usage
Input projection|.
:---|:---
`row`   | represents the genes
`col`   | represents the samples
`y-axis`             | is the input data for which to normalization


Output relations|.
:---|:---
`norm`| numeric, the normalized values of the measurements (i.e. of each sample)

##### Details
The operator is the `justvsn` function from the VSN bioconductor package.


##### References
see the `vsn` R package for the documentation.


##### See Also


##### Examples


##### GitHub link

[vsn_operator on GitHub](https://github.com/tercen/vsn_operator)