<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>RNA sequencing | Tercen operator catalog</title>
  <meta name="description" content="This is a compilation of Tercen operators" />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="RNA sequencing | Tercen operator catalog" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a compilation of Tercen operators" />
  <meta name="github-repo" content="tercen/operator_encyclopaedia" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="RNA sequencing | Tercen operator catalog" />
  
  <meta name="twitter:description" content="This is a compilation of Tercen operators" />
  

<meta name="author" content="Tercen team" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="flow-cytometry.html"/>
<link rel="next" href="bioinformatics.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<script src="libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<link href="libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="libs/datatables-binding-0.13/datatables.js"></script>
<link href="libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>
<link href="libs/crosstalk-1.1.0.1/css/crosstalk.css" rel="stylesheet" />
<script src="libs/crosstalk-1.1.0.1/js/crosstalk.min.js"></script>


<style type="text/css">
code.sourceCode > span { display: inline-block; line-height: 1.25; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface and operator index</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html"><i class="fa fa-check"></i>Basic statistics</a><ul>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#constant-operator"><i class="fa fa-check"></i>constant operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#count-operator"><i class="fa fa-check"></i>count operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#cv-operator"><i class="fa fa-check"></i>cv operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#density-operator"><i class="fa fa-check"></i>density operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#foldchange-operator"><i class="fa fa-check"></i>foldchange operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#iqr-operator"><i class="fa fa-check"></i>IQR operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#logfoldchange-operator"><i class="fa fa-check"></i>logfoldchange operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#max-operator"><i class="fa fa-check"></i>max operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#mean-imputation-operator"><i class="fa fa-check"></i>Mean imputation operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#mean-operator"><i class="fa fa-check"></i>mean operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#mean-sd-operator"><i class="fa fa-check"></i>mean sd operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#median-operator"><i class="fa fa-check"></i>Median operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#negative-operator"><i class="fa fa-check"></i>negative operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#pairwise-rmsd-operator"><i class="fa fa-check"></i>Pairwise RMSD operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#pearson-operator"><i class="fa fa-check"></i>pearson operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#percentile-operator"><i class="fa fa-check"></i>Percentile operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#introduction"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#rmsd-operator"><i class="fa fa-check"></i>RMSD operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#sample-operator"><i class="fa fa-check"></i>Sample operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#scale-operator"><i class="fa fa-check"></i>scale operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#sd-operator"><i class="fa fa-check"></i>sd operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#seqlen-operator"><i class="fa fa-check"></i>Seqlen operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#shift-operator"><i class="fa fa-check"></i>Shift operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#signchange-operator"><i class="fa fa-check"></i>signchange operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#topn-operator"><i class="fa fa-check"></i>topn operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#upper-and-lower-bounds-operator"><i class="fa fa-check"></i>Upper and lower bounds operator</a></li>
<li class="chapter" data-level="" data-path="basic-statistics.html"><a href="basic-statistics.html#word-count-operator"><i class="fa fa-check"></i>Word count operator</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="data-wrangling.html"><a href="data-wrangling.html"><i class="fa fa-check"></i>Data wrangling</a><ul>
<li class="chapter" data-level="" data-path="data-wrangling.html"><a href="data-wrangling.html#add-operator"><i class="fa fa-check"></i>add operator</a></li>
<li class="chapter" data-level="" data-path="data-wrangling.html"><a href="data-wrangling.html#asinh-operator"><i class="fa fa-check"></i>asinh operator</a></li>
<li class="chapter" data-level="" data-path="data-wrangling.html"><a href="data-wrangling.html#base-change-operator"><i class="fa fa-check"></i>Base change operator</a></li>
<li class="chapter" data-level="" data-path="data-wrangling.html"><a href="data-wrangling.html#calcnormfactors-operator"><i class="fa fa-check"></i>calcnormfactors operator</a></li>
<li class="chapter" data-level="" data-path="data-wrangling.html"><a href="data-wrangling.html#downsample-operator"><i class="fa fa-check"></i>Downsample operator</a></li>
<li class="chapter" data-level="" data-path="data-wrangling.html"><a href="data-wrangling.html#impute-operator"><i class="fa fa-check"></i>impute operator</a></li>
<li class="chapter" data-level="" data-path="data-wrangling.html"><a href="data-wrangling.html#log-operator"><i class="fa fa-check"></i>log operator</a></li>
<li class="chapter" data-level="" data-path="data-wrangling.html"><a href="data-wrangling.html#normalyzer_operator"><i class="fa fa-check"></i>normalyzer_operator</a></li>
<li class="chapter" data-level="" data-path="data-wrangling.html"><a href="data-wrangling.html#orderby-operator"><i class="fa fa-check"></i>orderBy operator</a></li>
<li class="chapter" data-level="" data-path="data-wrangling.html"><a href="data-wrangling.html#product-operator"><i class="fa fa-check"></i>product operator</a></li>
<li class="chapter" data-level="" data-path="data-wrangling.html"><a href="data-wrangling.html#random-label-operator"><i class="fa fa-check"></i>Random label operator</a></li>
<li class="chapter" data-level="" data-path="data-wrangling.html"><a href="data-wrangling.html#ratio-operator"><i class="fa fa-check"></i>ratio operator</a></li>
<li class="chapter" data-level="" data-path="data-wrangling.html"><a href="data-wrangling.html#replace-operator"><i class="fa fa-check"></i>Replace operator</a></li>
<li class="chapter" data-level="" data-path="data-wrangling.html"><a href="data-wrangling.html#round-operator"><i class="fa fa-check"></i>Round operator</a></li>
<li class="chapter" data-level="" data-path="data-wrangling.html"><a href="data-wrangling.html#separate-operator"><i class="fa fa-check"></i>Separate operator</a></li>
<li class="chapter" data-level="" data-path="data-wrangling.html"><a href="data-wrangling.html#subtract-operator"><i class="fa fa-check"></i>subtract operator</a></li>
<li class="chapter" data-level="" data-path="data-wrangling.html"><a href="data-wrangling.html#sum-operator"><i class="fa fa-check"></i>sum operator</a></li>
<li class="chapter" data-level="" data-path="data-wrangling.html"><a href="data-wrangling.html#vsn-operator"><i class="fa fa-check"></i>vsn operator</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="file-import.html"><a href="file-import.html"><i class="fa fa-check"></i>File import</a><ul>
<li class="chapter" data-level="" data-path="file-import.html"><a href="file-import.html#annotateddataframe-operator"><i class="fa fa-check"></i>AnnotatedDataFrame operator</a></li>
<li class="chapter" data-level="" data-path="file-import.html"><a href="file-import.html#readrds-operator"><i class="fa fa-check"></i>readRDS operator</a></li>
<li class="chapter" data-level="" data-path="file-import.html"><a href="file-import.html#read-excel-file-operator"><i class="fa fa-check"></i>Read excel file operator</a></li>
<li class="chapter" data-level="" data-path="file-import.html"><a href="file-import.html#read-fasta-operator"><i class="fa fa-check"></i>Read FASTA operator</a></li>
<li class="chapter" data-level="" data-path="file-import.html"><a href="file-import.html#fcs-zip-operator"><i class="fa fa-check"></i>FCS ZIP operator</a></li>
<li class="chapter" data-level="" data-path="file-import.html"><a href="file-import.html#read-fcs-operator"><i class="fa fa-check"></i>Read FCS operator</a></li>
<li class="chapter" data-level="" data-path="file-import.html"><a href="file-import.html#read-gmt-operator"><i class="fa fa-check"></i>Read GMT operator</a></li>
<li class="chapter" data-level="" data-path="file-import.html"><a href="file-import.html#straf-to-tercen-operator"><i class="fa fa-check"></i>STRAF to Tercen operator</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="statistical-testing.html"><a href="statistical-testing.html"><i class="fa fa-check"></i>Statistical testing</a><ul>
<li class="chapter" data-level="" data-path="statistical-testing.html"><a href="statistical-testing.html#anova-operator"><i class="fa fa-check"></i>ANOVA operator</a></li>
<li class="chapter" data-level="" data-path="statistical-testing.html"><a href="statistical-testing.html#lm-operator"><i class="fa fa-check"></i>lm operator</a></li>
<li class="chapter" data-level="" data-path="statistical-testing.html"><a href="statistical-testing.html#t-test-operator"><i class="fa fa-check"></i>t-test operator</a></li>
<li class="chapter" data-level="" data-path="statistical-testing.html"><a href="statistical-testing.html#two-group-t-test-operator"><i class="fa fa-check"></i>Two group t-test operator</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="data-visualisation.html"><a href="data-visualisation.html"><i class="fa fa-check"></i>Data visualisation</a><ul>
<li class="chapter" data-level="" data-path="data-visualisation.html"><a href="data-visualisation.html#boxplot-shiny-operator-for-tercen"><i class="fa fa-check"></i>Boxplot Shiny operator for Tercen</a></li>
<li class="chapter" data-level="" data-path="data-visualisation.html"><a href="data-visualisation.html#hierarchical-clustering-tree-operator"><i class="fa fa-check"></i>Hierarchical clustering tree operator</a></li>
<li class="chapter" data-level="" data-path="data-visualisation.html"><a href="data-visualisation.html#shiny-heatmap-tercen-operator"><i class="fa fa-check"></i>Shiny Heatmap Tercen operator</a></li>
<li class="chapter" data-level="" data-path="data-visualisation.html"><a href="data-visualisation.html#worldmap-docker-operator"><i class="fa fa-check"></i>Worldmap docker operator</a></li>
<li class="chapter" data-level="" data-path="data-visualisation.html"><a href="data-visualisation.html#shiny-world-map-tercen-operator"><i class="fa fa-check"></i>Shiny World Map Tercen operator</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="dimensionality-reduction-and-clustering.html"><a href="dimensionality-reduction-and-clustering.html"><i class="fa fa-check"></i>Dimensionality reduction and clustering</a><ul>
<li class="chapter" data-level="" data-path="dimensionality-reduction-and-clustering.html"><a href="dimensionality-reduction-and-clustering.html#approximated-t-sne"><i class="fa fa-check"></i>Approximated t-SNE</a></li>
<li class="chapter" data-level="" data-path="dimensionality-reduction-and-clustering.html"><a href="dimensionality-reduction-and-clustering.html#clustering-metrics-operator"><i class="fa fa-check"></i>Clustering metrics operator</a></li>
<li class="chapter" data-level="" data-path="dimensionality-reduction-and-clustering.html"><a href="dimensionality-reduction-and-clustering.html#hierarchical-clustering-tree-operator-1"><i class="fa fa-check"></i>Hierarchical clustering tree operator</a></li>
<li class="chapter" data-level="" data-path="dimensionality-reduction-and-clustering.html"><a href="dimensionality-reduction-and-clustering.html#clusterx-operator"><i class="fa fa-check"></i>clusterx operator</a></li>
<li class="chapter" data-level="" data-path="dimensionality-reduction-and-clustering.html"><a href="dimensionality-reduction-and-clustering.html#clusterx-operator-1"><i class="fa fa-check"></i>clusterx operator</a></li>
<li class="chapter" data-level="" data-path="dimensionality-reduction-and-clustering.html"><a href="dimensionality-reduction-and-clustering.html#fast-t-sne-docker-operator"><i class="fa fa-check"></i>Fast t-SNE Docker Operator</a></li>
<li class="chapter" data-level="" data-path="dimensionality-reduction-and-clustering.html"><a href="dimensionality-reduction-and-clustering.html#fast-t-sne-operator"><i class="fa fa-check"></i>Fast t-SNE operator</a></li>
<li class="chapter" data-level="" data-path="dimensionality-reduction-and-clustering.html"><a href="dimensionality-reduction-and-clustering.html#hclust-operator"><i class="fa fa-check"></i>hclust operator</a></li>
<li class="chapter" data-level="" data-path="dimensionality-reduction-and-clustering.html"><a href="dimensionality-reduction-and-clustering.html#mds-operator"><i class="fa fa-check"></i>MDS operator</a></li>
<li class="chapter" data-level="" data-path="dimensionality-reduction-and-clustering.html"><a href="dimensionality-reduction-and-clustering.html#flowsom-operator"><i class="fa fa-check"></i>flowsom operator</a></li>
<li class="chapter" data-level="" data-path="dimensionality-reduction-and-clustering.html"><a href="dimensionality-reduction-and-clustering.html#pca-operator"><i class="fa fa-check"></i>pca operator</a></li>
<li class="chapter" data-level="" data-path="dimensionality-reduction-and-clustering.html"><a href="dimensionality-reduction-and-clustering.html#rphenograph-operator"><i class="fa fa-check"></i>rphenograph operator</a></li>
<li class="chapter" data-level="" data-path="dimensionality-reduction-and-clustering.html"><a href="dimensionality-reduction-and-clustering.html#somflow-operator"><i class="fa fa-check"></i>somflow operator</a></li>
<li class="chapter" data-level="" data-path="dimensionality-reduction-and-clustering.html"><a href="dimensionality-reduction-and-clustering.html#tsne-operator"><i class="fa fa-check"></i>tsne operator</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="flow-cytometry.html"><a href="flow-cytometry.html"><i class="fa fa-check"></i>Flow Cytometry</a><ul>
<li class="chapter" data-level="" data-path="flow-cytometry.html"><a href="flow-cytometry.html#diffcyt-operator"><i class="fa fa-check"></i>diffcyt operator</a></li>
<li class="chapter" data-level="" data-path="flow-cytometry.html"><a href="flow-cytometry.html#flowai-operator"><i class="fa fa-check"></i>flowai operator</a></li>
<li class="chapter" data-level="" data-path="flow-cytometry.html"><a href="flow-cytometry.html#flowsom-tuning-operator"><i class="fa fa-check"></i>FlowSOM tuning operator</a></li>
<li class="chapter" data-level="" data-path="flow-cytometry.html"><a href="flow-cytometry.html#flowsom-operator-1"><i class="fa fa-check"></i>FlowSOM operator</a></li>
<li class="chapter" data-level="" data-path="flow-cytometry.html"><a href="flow-cytometry.html#mem-operator"><i class="fa fa-check"></i>MEM operator</a></li>
<li class="chapter" data-level="" data-path="flow-cytometry.html"><a href="flow-cytometry.html#neighbourhood-operator"><i class="fa fa-check"></i>neighbourhood operator</a></li>
<li class="chapter" data-level="" data-path="flow-cytometry.html"><a href="flow-cytometry.html#flowsom-operator-2"><i class="fa fa-check"></i>flowsom operator</a></li>
<li class="chapter" data-level="" data-path="flow-cytometry.html"><a href="flow-cytometry.html#rphenograph-operator-1"><i class="fa fa-check"></i>rphenograph operator</a></li>
<li class="chapter" data-level="" data-path="flow-cytometry.html"><a href="flow-cytometry.html#somflow-operator-1"><i class="fa fa-check"></i>somflow operator</a></li>
<li class="chapter" data-level="" data-path="flow-cytometry.html"><a href="flow-cytometry.html#umap-operator"><i class="fa fa-check"></i>umap operator</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="rna-sequencing.html"><a href="rna-sequencing.html"><i class="fa fa-check"></i>RNA sequencing</a><ul>
<li class="chapter" data-level="" data-path="rna-sequencing.html"><a href="rna-sequencing.html#calcnormfactors-operator-1"><i class="fa fa-check"></i>calcnormfactors operator</a></li>
<li class="chapter" data-level="" data-path="rna-sequencing.html"><a href="rna-sequencing.html#deseq2_two_conditions-operator"><i class="fa fa-check"></i>DESeq2_two_conditions operator</a></li>
<li class="chapter" data-level="" data-path="rna-sequencing.html"><a href="rna-sequencing.html#diffcyt-operator-1"><i class="fa fa-check"></i>diffcyt operator</a></li>
<li class="chapter" data-level="" data-path="rna-sequencing.html"><a href="rna-sequencing.html#fgsea-operator"><i class="fa fa-check"></i>fgsea operator</a></li>
<li class="chapter" data-level="" data-path="rna-sequencing.html"><a href="rna-sequencing.html#flowsom-tuning-operator-1"><i class="fa fa-check"></i>FlowSOM tuning operator</a></li>
<li class="chapter" data-level="" data-path="rna-sequencing.html"><a href="rna-sequencing.html#umap-operator-1"><i class="fa fa-check"></i>umap operator</a></li>
<li class="chapter" data-level="" data-path="rna-sequencing.html"><a href="rna-sequencing.html#vsn-operator-1"><i class="fa fa-check"></i>vsn operator</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="bioinformatics.html"><a href="bioinformatics.html"><i class="fa fa-check"></i>Bioinformatics</a><ul>
<li class="chapter" data-level="" data-path="bioinformatics.html"><a href="bioinformatics.html#alignment-metrics-operator"><i class="fa fa-check"></i>Alignment metrics operator</a></li>
<li class="chapter" data-level="" data-path="bioinformatics.html"><a href="bioinformatics.html#pairwise-sequence-distance-operator"><i class="fa fa-check"></i>Pairwise sequence distance operator</a></li>
<li class="chapter" data-level="" data-path="bioinformatics.html"><a href="bioinformatics.html#gene-set-enrichment-analysis-operator"><i class="fa fa-check"></i>Gene Set Enrichment Analysis operator</a></li>
<li class="chapter" data-level="" data-path="bioinformatics.html"><a href="bioinformatics.html#multiple-sequence-alignment-operator"><i class="fa fa-check"></i>Multiple Sequence Alignment operator</a></li>
<li class="chapter" data-level="" data-path="bioinformatics.html"><a href="bioinformatics.html#msa-shiny-operator"><i class="fa fa-check"></i>MSA Shiny operator</a></li>
<li class="chapter" data-level="" data-path="bioinformatics.html"><a href="bioinformatics.html#read-fasta-operator-1"><i class="fa fa-check"></i>Read FASTA operator</a></li>
<li class="chapter" data-level="" data-path="bioinformatics.html"><a href="bioinformatics.html#sequence-to-table-operator"><i class="fa fa-check"></i>Sequence to table operator</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="pharmacokinetics.html"><a href="pharmacokinetics.html"><i class="fa fa-check"></i>Pharmacokinetics</a><ul>
<li class="chapter" data-level="" data-path="pharmacokinetics.html"><a href="pharmacokinetics.html#base-change-operator-1"><i class="fa fa-check"></i>Base change operator</a></li>
<li class="chapter" data-level="" data-path="pharmacokinetics.html"><a href="pharmacokinetics.html#egfr-operator"><i class="fa fa-check"></i>eGFR operator</a></li>
<li class="chapter" data-level="" data-path="pharmacokinetics.html"><a href="pharmacokinetics.html#non-compartmental-pk-modelling"><i class="fa fa-check"></i>Non Compartmental PK Modelling</a></li>
<li class="chapter" data-level="" data-path="pharmacokinetics.html"><a href="pharmacokinetics.html#snca-operator"><i class="fa fa-check"></i>sNCA operator</a></li>
<li class="chapter" data-level="" data-path="pharmacokinetics.html"><a href="pharmacokinetics.html#upper-and-lower-bounds-operator-1"><i class="fa fa-check"></i>Upper and lower bounds operator</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="forensics.html"><a href="forensics.html"><i class="fa fa-check"></i>Forensics</a><ul>
<li class="chapter" data-level="" data-path="forensics.html"><a href="forensics.html#forensic-parameters-operator"><i class="fa fa-check"></i>Forensic parameters operator</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="template.html"><a href="template.html"><i class="fa fa-check"></i>Template</a><ul>
<li class="chapter" data-level="" data-path="template.html"><a href="template.html#shiny-docker-operator"><i class="fa fa-check"></i>Shiny docker operator</a></li>
<li class="chapter" data-level="" data-path="template.html"><a href="template.html#template-docker-operator"><i class="fa fa-check"></i>Template Docker Operator</a></li>
<li class="chapter" data-level="" data-path="template.html"><a href="template.html#template-r-operator-for-tercen"><i class="fa fa-check"></i>Template R operator for Tercen</a></li>
<li class="chapter" data-level="" data-path="template.html"><a href="template.html#template-shiny-operator-for-tercen"><i class="fa fa-check"></i>Template Shiny operator for Tercen</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Tercen operator catalog</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="rna-sequencing" class="section level1">
<h1>RNA sequencing</h1>
<div id="calcnormfactors-operator-1" class="section level2">
<h2>calcnormfactors operator</h2>
<div id="description-83" class="section level5">
<h5>Description</h5>
<p><code>calcnormfactors</code> operator calculates a normalization factor per column (i.e sample).</p>
</div>
<div id="usage-81" class="section level5">
<h5>Usage</h5>
<table>
<thead>
<tr class="header">
<th align="left">Input projection</th>
<th align="left">.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><code>row</code></td>
<td align="left">represents the genes</td>
</tr>
<tr class="even">
<td align="left"><code>col</code></td>
<td align="left">represents the samples</td>
</tr>
<tr class="odd">
<td align="left"><code>y-axis</code></td>
<td align="left">is the input data for which to compute the normalization factor</td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr class="header">
<th align="left">Output relations</th>
<th align="left">.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><code>normfactor</code></td>
<td align="left">numeric, per column (i.e. per sample)</td>
</tr>
</tbody>
</table>
</div>
<div id="details-75" class="section level5">
<h5>Details</h5>
<p>The operator is the calcnormfactor in the edgeR bioconductor.</p>
</div>
<div id="references-61" class="section level5">
<h5>References</h5>
<p>see the <code>edgeR</code> R package for the documentation.</p>
</div>
<div id="see-also-65" class="section level5">
<h5>See Also</h5>
</div>
<div id="examples-40" class="section level5">
<h5>Examples</h5>
</div>
<div id="github-link-86" class="section level5">
<h5>GitHub link</h5>
<p><a href="https://github.com/tercen/calcnormfactors_operator">calcnormfactors_operator on GitHub</a></p>
</div>
</div>
<div id="deseq2_two_conditions-operator" class="section level2">
<h2>DESeq2_two_conditions operator</h2>
<div id="description-84" class="section level5">
<h5>Description</h5>
<p><code>DESeq2_two_conditions</code> tests for differential gene expression in samples from two conditions using the <code>DESeq2</code> package from BioConductor (Love, et al, Genome Biology, 2014).</p>
</div>
<div id="usage-82" class="section level5">
<h5>Usage</h5>
<table>
<thead>
<tr class="header">
<th align="left">Input projection</th>
<th align="left">Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><code>row</code></td>
<td align="left">Gene name/identifier</td>
</tr>
<tr class="even">
<td align="left"><code>column</code></td>
<td align="left">Sample name/identifier</td>
</tr>
<tr class="odd">
<td align="left"><code>color</code></td>
<td align="left">Represents the groups to compare</td>
</tr>
<tr class="even">
<td align="left"><code>y-axis</code></td>
<td align="left">Sequence counts</td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="16%" />
<col width="83%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Input parameters</th>
<th align="left">Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><code>alpha</code></td>
<td align="left">Numeric, adjusted <em>p value</em> cutoff for independent filtering (default = 0.1)</td>
</tr>
<tr class="even">
<td align="left"><code>LFC_shrinkage</code></td>
<td align="left">Logical, whether the returned log fold-change values should be shrinked (default = TRUE)</td>
</tr>
<tr class="odd">
<td align="left"><code>shrinkage_type</code></td>
<td align="left">“normal”, “apeglm” or “ashr”, Type of shrinkage estimator to use (default = “normal”)</td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="20%" />
<col width="79%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Output relations</th>
<th align="left">Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><code>pvalue</code></td>
<td align="left">numeric, p-value calculated per gene</td>
</tr>
<tr class="even">
<td align="left"><code>padj</code></td>
<td align="left">numeric, p-value calculated per gene after adjusting for multiple testing</td>
</tr>
<tr class="odd">
<td align="left"><code>baseMean</code></td>
<td align="left">numeric, mean of normalized counts for all the samples</td>
</tr>
<tr class="even">
<td align="left"><code>log2FoldChange</code></td>
<td align="left">numeric, shrunken log2 fold-change between the two groups to compare</td>
</tr>
<tr class="odd">
<td align="left"><code>minus_log10_padj</code></td>
<td align="left">numeric, negative log10 transformation of padj for more intuitive plotting</td>
</tr>
</tbody>
</table>
</div>
<div id="details-76" class="section level5">
<h5>Details</h5>
<p>The operator uses the <code>DESeq2</code> package from BioConductor.</p>
</div>
<div id="references-62" class="section level5">
<h5>References</h5>
<p>Love MI, Huber W, Anders S (2014). “Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2.” Genome Biology, 15, 550. doi: 10.1186/s13059-014-0550-8.</p>
<p>See <a href="https://bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html">“Analyzing RNA-seq data with DESeq2”</a> for further information on DESeq2 by Love, et al.</p>
</div>
<div id="github-link-87" class="section level5">
<h5>GitHub link</h5>
<p><a href="https://github.com/tercen/DESeq2_two_conditions_operator">DESeq2_two_conditions_operator on GitHub</a></p>
</div>
</div>
<div id="diffcyt-operator-1" class="section level2">
<h2>diffcyt operator</h2>
<div id="description-85" class="section level5">
<h5>Description</h5>
<p><code>diffcyt</code> operator a differential analysis of flow cyto data and indicates which marker and cluster combinations are relevant.</p>
</div>
<div id="usage-83" class="section level5">
<h5>Usage</h5>
<table>
<thead>
<tr class="header">
<th align="left">Input projection</th>
<th align="left">.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><code>col</code></td>
<td align="left"><code>group_id</code>, <code>patient_id</code></td>
</tr>
<tr class="even">
<td align="left"><code>row</code></td>
<td align="left"><code>marker_class</code>, <code>marker_name</code></td>
</tr>
<tr class="odd">
<td align="left"><code>y-axis</code></td>
<td align="left">values representing measurement</td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="50%" />
<col width="50%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Input parameters</th>
<th align="left">.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><code>analysis_type</code></td>
<td align="left">can be either <code>DA</code> (Differential Abundance) or <code>DS</code> (Differential State)</td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr class="header">
<th align="left">Output relations</th>
<th align="left">.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><code>cluster_id</code></td>
<td align="left">character, cluster name, per cluster, DA and DS output</td>
</tr>
<tr class="even">
<td align="left"><code>LogFC</code></td>
<td align="left">numeric, log fold change, per cluster, DA</td>
</tr>
<tr class="odd">
<td align="left"><code>LR</code></td>
<td align="left">numeric, lr, per cluster, DA output</td>
</tr>
<tr class="even">
<td align="left"><code>p_val</code></td>
<td align="left">numeric, p value, per cluster, DA output</td>
</tr>
<tr class="odd">
<td align="left"><code>p_adj</code></td>
<td align="left">numeric, adjusted p value, per cluster, DA output</td>
</tr>
<tr class="even">
<td align="left"><code>LogFC</code></td>
<td align="left">numeric, log fold change, per cluster-marker, DS output</td>
</tr>
<tr class="odd">
<td align="left"><code>p_val</code></td>
<td align="left">numeric, p value, per cluster-marker, DS output</td>
</tr>
<tr class="even">
<td align="left"><code>p_adj</code></td>
<td align="left">numeric, adjusted p value, per cluster-marker, DS output</td>
</tr>
<tr class="odd">
<td align="left"><code>t</code></td>
<td align="left">numeric, t, per cluster-marker, DS output</td>
</tr>
<tr class="even">
<td align="left"><code>B</code></td>
<td align="left">numeric, B, per cluster-marker, DS output</td>
</tr>
<tr class="odd">
<td align="left"><code>AvgExp</code></td>
<td align="left">numeric, adjusted p value, per cluster-marker, DS output</td>
</tr>
</tbody>
</table>
</div>
<div id="details-77" class="section level5">
<h5>Details</h5>
<p>Performs differential analysis (abundance or state). See the <code>diffcyt::diffcyt</code> function in the Bioconductor R pacakge.</p>
</div>
<div id="references-63" class="section level5">
<h5>References</h5>
<p>see the github for documentation,
<a href="https://github.com/lmweber/diffcyt" class="uri">https://github.com/lmweber/diffcyt</a></p>
</div>
<div id="see-also-66" class="section level5">
<h5>See Also</h5>
<p><a href="https://github.com/lmweber/ttest_operator">t-test</a>, <a href="https://github.com/tercen/anova_operator">anova</a>, <a href="https://github.com/tercen/rfImp_operator">rfImp</a></p>
</div>
<div id="examples-41" class="section level5">
<h5>Examples</h5>
</div>
<div id="github-link-88" class="section level5">
<h5>GitHub link</h5>
<p><a href="https://github.com/tercen/diffcyt_operator">diffcyt_operator on GitHub</a></p>
</div>
</div>
<div id="fgsea-operator" class="section level2">
<h2>fgsea operator</h2>
<div id="description-86" class="section level5">
<h5>Description</h5>
<p><code>fgsea</code> operator performs a fast gene set enrichment analysis.</p>
</div>
<div id="usage-84" class="section level5">
<h5>Usage</h5>
<table>
<thead>
<tr class="header">
<th align="left">Input projection</th>
<th align="left">.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><code>y-axis</code></td>
<td align="left">numeric, input data usually a ranking statistics, per cell</td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr class="header">
<th align="left">Output relations</th>
<th align="left">.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><code>NES</code></td>
<td align="left">numeric, fgsea of the input data</td>
</tr>
<tr class="even">
<td align="left"><code>padj</code></td>
<td align="left">numeric, adjusted p-value</td>
</tr>
</tbody>
</table>
</div>
<div id="details-78" class="section level5">
<h5>Details</h5>
<p>The operator takes all the values of a cell and returns the value which is the fgsea. The computation is done per cell. There is one value returned for each of the input cell.</p>
</div>
<div id="references-64" class="section level5">
<h5>References</h5>
</div>
<div id="see-also-67" class="section level5">
<h5>See Also</h5>
</div>
<div id="examples-42" class="section level5">
<h5>Examples</h5>
</div>
<div id="github-link-89" class="section level5">
<h5>GitHub link</h5>
<p><a href="https://github.com/tercen/fgsea_operator">fgsea_operator on GitHub</a></p>
</div>
</div>
<div id="flowsom-tuning-operator-1" class="section level2">
<h2>FlowSOM tuning operator</h2>
<div id="description-87" class="section level5">
<h5>Description</h5>
<p><code>flowsomtuning</code> operator performs flowSOM clustering for different numbers of clusters.</p>
</div>
<div id="usage-85" class="section level5">
<h5>Usage</h5>
<table>
<thead>
<tr class="header">
<th align="left">Input projection</th>
<th align="left">.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><code>row</code></td>
<td align="left">represents the variables (e.g. channels, markers)</td>
</tr>
<tr class="even">
<td align="left"><code>col</code></td>
<td align="left">represents the clusters (e.g. cells)</td>
</tr>
<tr class="odd">
<td align="left"><code>y-axis</code></td>
<td align="left">is the value of measurement signal of the channel/marker</td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr class="header">
<th align="left">Input parameters</th>
<th align="left">.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><code>min_cluster_number</code></td>
<td align="left">Minimum number of clusters to make</td>
</tr>
<tr class="even">
<td align="left"><code>max_cluster_number</code></td>
<td align="left">Maximal number of clusters to make</td>
</tr>
<tr class="odd">
<td align="left"><code>transform</code></td>
<td align="left">Transform data?</td>
</tr>
<tr class="even">
<td align="left"><code>seed</code></td>
<td align="left">Random seed</td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr class="header">
<th align="left">Output relations</th>
<th align="left">.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><code>cluster</code></td>
<td align="left">character, cluster label</td>
</tr>
</tbody>
</table>
</div>
<div id="details-79" class="section level5">
<h5>Details</h5>
<p>The operator is a wrapper for the <code>FlowSOM</code> function of the <code>FlowSOM</code> R/Bioconductor package.</p>
</div>
<div id="references-65" class="section level5">
<h5>References</h5>
<p><a href="https://bioconductor.org/packages/FlowSOM/" class="uri">https://bioconductor.org/packages/FlowSOM/</a></p>
</div>
<div id="see-also-68" class="section level5">
<h5>See Also</h5>
<p><a href="https://github.com/tercen/flowsom_operator">flowsom_operator</a></p>
<p><a href="https://github.com/tercen/flowsom_mst_shiny_operator">flowsom_mst_shiny_operator</a></p>
</div>
<div id="github-link-90" class="section level5">
<h5>GitHub link</h5>
<p><a href="https://github.com/tercen/flowsomtuning_operator">flowsomtuning_operator on GitHub</a></p>
</div>
</div>
<div id="umap-operator-1" class="section level2">
<h2>umap operator</h2>
<div id="description-88" class="section level5">
<h5>Description</h5>
<p><code>umap</code> operator performs umap analysis.</p>
</div>
<div id="usage-86" class="section level5">
<h5>Usage</h5>
<table>
<thead>
<tr class="header">
<th align="left">Input projection</th>
<th align="left">.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><code>row</code></td>
<td align="left">represents the variables (e.g. genes, channels, markers)</td>
</tr>
<tr class="even">
<td align="left"><code>col</code></td>
<td align="left">represents the observations (e.g. cells, samples, individuals)</td>
</tr>
<tr class="odd">
<td align="left"><code>y-axis</code></td>
<td align="left">measurement value</td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="50%" />
<col width="50%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Input parameters</th>
<th align="left">.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><code>init</code></td>
<td align="left">character, type of initialization for the coordinates, see details</td>
</tr>
<tr class="even">
<td align="left"><code>scale</code></td>
<td align="left">numeric, type of scaling to apply to data</td>
</tr>
<tr class="odd">
<td align="left"><code>spread</code></td>
<td align="left">numeric, the effective scale of embedded points. In combination with <code>min_dist</code>, this determines how clustered/clumped the embedded points are</td>
</tr>
<tr class="even">
<td align="left"><code>min_dist</code></td>
<td align="left">numeric, the effective minimum distance between embedded point</td>
</tr>
<tr class="odd">
<td align="left"><code>pca</code></td>
<td align="left">numeric, If set to a positive integer value, reduce data to this number of columns using PCA</td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr class="header">
<th align="left">Output relations</th>
<th align="left">.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><code>umap01, umap02</code></td>
<td align="left">first two components containing the new projected values</td>
</tr>
</tbody>
</table>
</div>
<div id="details-80" class="section level5">
<h5>Details</h5>
<p>The operator performs umap analysis. It reduces the amount of variables (i.e. indicated by rows) to a lower number (default 2). This operators wraps the <code>uwot::umap()</code>. See (<a href="https://github.com/jlmelville/uwot" class="uri">https://github.com/jlmelville/uwot</a>) for more details, especially settings and examples.</p>
</div>
<div id="reference-5" class="section level5">
<h5>Reference</h5>
</div>
<div id="see-also-69" class="section level5">
<h5>See Also</h5>
<p><a href="https://github.com/tercen/pca_operator">pca</a>, <a href="https://github.com/tercen/tsne_operator">tsne</a></p>
</div>
<div id="examples-43" class="section level5">
<h5>Examples</h5>
</div>
<div id="github-link-91" class="section level5">
<h5>GitHub link</h5>
<p><a href="https://github.com/tercen/umap_operator">umap_operator on GitHub</a></p>
</div>
</div>
<div id="vsn-operator-1" class="section level2">
<h2>vsn operator</h2>
<div id="description-89" class="section level5">
<h5>Description</h5>
<p><code>vsn</code> operator performs a normalization factor per column (i.e. sample).</p>
</div>
<div id="usage-87" class="section level5">
<h5>Usage</h5>
<table>
<thead>
<tr class="header">
<th align="left">Input projection</th>
<th align="left">.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><code>row</code></td>
<td align="left">represents the genes</td>
</tr>
<tr class="even">
<td align="left"><code>col</code></td>
<td align="left">represents the samples</td>
</tr>
<tr class="odd">
<td align="left"><code>y-axis</code></td>
<td align="left">is the input data for which to normalization</td>
</tr>
</tbody>
</table>
<table>
<colgroup>
<col width="50%" />
<col width="50%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Output relations</th>
<th align="left">.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><code>norm</code></td>
<td align="left">numeric, the normalized values of the measurements (i.e. of each sample)</td>
</tr>
</tbody>
</table>
</div>
<div id="details-81" class="section level5">
<h5>Details</h5>
<p>The operator is the <code>justvsn</code> function from the VSN bioconductor package.</p>
</div>
<div id="references-66" class="section level5">
<h5>References</h5>
<p>see the <code>vsn</code> R package for the documentation.</p>
</div>
<div id="see-also-70" class="section level5">
<h5>See Also</h5>
</div>
<div id="examples-44" class="section level5">
<h5>Examples</h5>
</div>
<div id="github-link-92" class="section level5">
<h5>GitHub link</h5>
<p><a href="https://github.com/tercen/vsn_operator">vsn_operator on GitHub</a></p>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="flow-cytometry.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="bioinformatics.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": null,
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"search": true,
"info": true
});
});
</script>

</body>

</html>
